README.md

```diff
@@ -72,14 +72,39 @@ sonar.pullrequest.base = target_branch_name (e.g master)
 :warning: There must not be any `sonar.branch` properties like `sonar.branch.name` arguments set when you analyze a
   pull-request. These properties indicate to sonar that a branch is being analyzed rather than a pull-request so no
     pull-request decoration will be executed.
-    
+
 ## Serving images for PR decoration
-By default, images for PR decoration are served as static resources on the SonarQube server as a part of Community Branch Plugin. 
+By default, images for PR decoration are served as static resources on the SonarQube server as a part of Community Branch Plugin.
 
 If you use a SonarQube server behind a firewall and/or PR service (Github, Gitlab etc.) hasn't access to SonarQube server, you should change `Images base URL` property in `General > Pull Request` settings.
 
 Anyone needing to set this value can use the URL `https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/master/src/main/resources/static`, or download the files from this location and host them themself.
- 
+
 # Building the plugin from source
 In case you want to try and test the current branch or build it for your development execute `./gradlew clean build
 ` inside of the project directory. This will put the built jar under `libs/sonarqube-community-branch-plugin*.jar`
+
+# Using the bitnami AMI image
+
+### Set up server from bitnami API
+- Spin up an EC2 instance with the bitnami AMI (https://aws.amazon.com/marketplace/pp/prodview-zweq32he6zjmw?applicationId=AWSMPContessa#pdp-overview)
+- Set up SSL (process not documented here)
+- Set up DNS (process not documented here)
+- SSH into server (process not documented here)
+
+### Once SSHed into machine
+- `$ sudo su`
+- `$ cd /opt/bitnami/sonarqube/extensions/plugins/`
+- `$ wget https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.11.0/sonarqube-community-branch-plugin-1.11.0.jar`
+-` $ chmod 777 sonarqube-community-branch-plugin-1.11.0.jar`
+- `$ chown bitnami sonarqube-community-branch-plugin-1.11.0.jar`
+- `$ chgrp sonarqube sonarqube-community-branch-plugin-1.11.0.jar`
```

Comment:
this could be simplifed to `chown bitnami:sonarqube sonarqube-community-branch-plugin-1.11.0.jar`, and shouldn't be necessary if we've `su`d to the right user at the start of the process.